src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,173 @@
+
+<aside class="alert alert-info" markdown="1">
+**Web libraries:**
+If you write web apps, you should know about Angular 2, a web application framework.
+You might try a [code lab for Angular Dart](https://webdev.dartlang.org/codelabs/ng2),
+or refer to the [Angular Dart docs](https://angular.io/docs/dart/latest/).
+For more information about Dart on the Web, see [webdev.dartlang]({{site.webdev}}).
+</aside>
+
+<aside class="alert alert-info" markdown="1">
+**Flutter libraries:**
+If you write mobile applications, you'll want to use Flutter.
+The core libraries distributed with the Flutter SDK are documented at
+[docs.flutter.io](http://docs.flutter.io/). To import these libraries,
+follow the instructions in this
+[guide](https://www.dartlang.org/tools/pub/get-started#importing-libraries-from-packages).
+Additional third-party libraries for Flutter can be found on GitHub.
+For information about how to use a shared library,
+see its README documentation on GitHub.
+For more information about writing mobile applications using Dart,
+see [flutter.io]({{site.flutter}}).
+</aside>
+
+<aside class="alert alert-info" markdown="1">
+**Dart VM libraries:**
+If you write servers or command-line applications, you'll
+want to use [dart:io](https://api.dartlang.org/stable/dart-io/dart-io-library.html)
+and related libraries, such as
+[http_server](https://pub.dartlang.org/packages/http_server).
+For more information, see [Dart VM](/dart-vm/).
+</aside>
+
+## Dart libraries and their corresponding packages
+
+Dart's core libraries contain classes most commonly used by developers.
+Classes that are still useful, but not as universally used,
+are placed in packages outside of the SDK.
+Separating out the less commonly used classes ensures that your app
+isn't bloated by features that you don't need.
+For example, the
+[dart:collection](https://api.dartlang.org/stable/dart-collection/dart-collection-library.html)
+library contains the Queue, HashMap, and HashSet classes. The
+[collection package](https://www.dartdocs.org/documentation/collection/latest/collection/collection-library.html)
+contains more specialized classes, such as CanonicalizedMap, MapKeySet, PriorityQueue,
+and support for equality testing.
+
+If you don't find the functionality you need on api.dartlang.org, check pub.dartlang.
+
+<aside class="alert alert-info" markdown="1">
+**Terminology:**
+Libraries and packages aren't interchangeable terms. Both libraries and
+applications are distributed as packages. A library can be part of a package,
+but a package is not a library.
+For more information on what constitutes a library package, see
+[Create Library Packages](/guides/libraries/create-library-packages).
+</aside>
+
+Some of the libraries in the SDK have corresponding packages with additional
+functionality. The following table includes some that you should know about:
+
+| **SDK Library** | **Package** |
```

Comment:
This is confusing. At first few glances, I thought it was one row per package, but it's SDK package --> pub package.

FWIW, I suggest you flatten the table into two.

The first table:

| **SDK Library** | **Description** | **Example members** |
...

Then the next:

| **Package** | **Description** | **Example members** |
...

You can have the 3rd party packages in the same vein. We can style it so that the column widths are consistent.

I know what you're trying to do here (SDK libraries + their extension) but it looks confusing and it breaks with dart:core and dart:js (which we shouldn't even mention).